packages/monaco/src/browser/monaco-editor-provider.ts

```diff
@@ -68,22 +72,29 @@ export class MonacoEditorProvider {
             commandService.setDelegate(standaloneCommandService);
 
             return editor;
-        })));
+        });
+    }
+
+    protected getModelOptions() {
+        return {
+            tabSize: this.editorPreferences["editor.tabSize"]
+        };
     }
 
     protected async getEditorOptions(model: MonacoEditorModel): Promise<MonacoEditor.IOptions | undefined> {
```

Comment:
You're right, I completely forgot to remove all the asynchronous methods. I think it should be fine now.